<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>character_form</class>
 <widget class="QWidget" name="character_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>498</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="character_form_layout">
   <item>
    <layout class="QVBoxLayout" name="character_list_layout">
     <item>
      <widget class="QListView" name="character_list">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="character_list_control_layout">
       <item>
        <widget class="QPushButton" name="character_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="character_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="character_detail_layout">
     <item row="0" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="description_label">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="description_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="selection_weight_label">
       <property name="text">
        <string>Selection Weight</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="selection_weight_spin">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="resource_label">
       <property name="text">
        <string>Resource</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="resource_combo"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="costumes_label">
       <property name="text">
        <string>Costumes</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QListView" name="costumes_list"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>character_add</sender>
   <signal>clicked()</signal>
   <receiver>character_form</receiver>
   <slot>add_character()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>character_remove</sender>
   <signal>clicked()</signal>
   <receiver>character_form</receiver>
   <slot>remove_characters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_character()</slot>
  <slot>remove_characters()</slot>
 </slots>
</ui>
